website/docs/r/vapp_vm.html.markdown

```diff
@@ -191,6 +214,20 @@ example for usage details. **Deprecates**: `network_name`, `ip`, `vapp_network_n
 
   * `ip_allocation_mode=NONE` - **`ip`** field can be omitted or set to an empty string "". Empty string may be useful when doing HCL variable interpolation.
 
+<a id="override_template_disk"></a>
+## Override template disk
+Allows to update internal disk in template before first VM boot. Disk are matched by `bus_type`, `bus_number` and `unit_number`
+Changes are ignored on update. To manage internal disk later please use [`vcd_vm_internal_disk`](/docs/providers/vcd/r/vm_internal_disk.html) resource. 
```

Comment:
Need a comment that VDC must be **not** fast provisioned for this to work.

Reply:
added

Reply:
Let's polish the phrasing to:
"Managing disks in VM is possible only when VDC fast provisioned is disabled."
Also, as it is pretty important limitation, what about putting it into a yellow note block?

Reply:
done